sunpy/coordinates/__init__.py

```diff
@@ -1,7 +1,29 @@
+"""
+This subpackage contains:
+
+* A robust framework for working with coordinate systems
+* Functions to obtain the locations of solar-system bodies
+* Functions to calculate Sun-specific coordinate information
+  (`sunpy.coordinates.sun`)
+
+The diagram below shows all of Sun-based and Earth-based coordinate systems
+available through `sunpy.coordinates`, as well as the transformations between
+them.  Each frame is labeled with the name of its class and its alias (useful
+for converting other coordinates to them using attribute-style access).
+
+The frames colored in cyan are implemented in `astropy.coordinates`, and there
+are other astronomical frames that can be transformed to that are not shown
+below.  See the documentation for `astropy.coordinates` for more information.
+
+"""
+
 from .frames import *
 from .offset_frame import *
 from . import transformations
+from .transformations import _make_sunpy_graph
```

Comment:
Why import this like this when we are already importing `transformations` above?

Reply:
Because I can't read. =P  Fixed.